ops/terraform/env/mgmt/iam.tf

```diff
@@ -1,90 +1,25 @@
-#TODO: Determine if the bfd-packages sees continued use
-resource "aws_iam_policy" "packer_s3" {
```

Comment:
This policy definition was fully removed. Answering the above TODO: the `bfd-packages` S3 bucket at issue does not exist and has not existed any time recently.